main/src/vtr_common/vtr_include.cmake

```diff
@@ -18,6 +18,7 @@ add_compile_options(-march=native -O3 -pthread -Wall -Wextra)
 # address sanitizer
 # add_compile_options(-fsanitize=address)
 # set(CMAKE_CXX_STANDARD_LIBRARIES -lasan)
+add_compile_options(-g -Og)
```

Comment:
Please reset this change.
We decided that the default will be to compile without debugging symbols on main.
(It should just be commented out)

Reply:
Fixed